packages/grid/helm/syft/values.yaml

```diff
@@ -159,15 +159,20 @@ node:
   rootEmail: info@openmined.org
   type: domain
   side: high
-  inMemoryWorkers: false
-  defaultWorkerPoolCount: 1
   defaultBucketName: null
+  inMemoryWorkers: false
   queuePort: 5556
   logLevel: info
   debuggerEnabled: false
   associationRequestAutoApproval: false
   useInternalRegistry: true
 
+  # Deault Worker pool settings
+  defaultWorkerPool:
```

Comment:
@rasswanth-s  we also need to update the following yaml files:
- devspace.yaml
- helm/examples/**/*.yaml
- values.dev.*.yaml

Reply:
Addressed in [a33d981](https://github.com/OpenMined/PySyft/pull/8884/commits/a33d981f486457ec26601d194d90cc5a638f9fa8)